server/data/people.js

```diff
@@ -30,12 +30,13 @@ export const chrisChapman: Person = {
 };
 
 export const robertBidder: Person = {
-  givenName: 'Robert',
+  givenName: 'Rob',
   familyName: 'Bidder',
   name: 'Rob Bidder',
   twitterHandle: 'robertbidder',
-  description: 'Robert is a Visitor Experience Assistant at Wellcome Collection. When not in the museum he is making art and music at home.',
-  image: 'https://wellcomecollection.files.wordpress.com/2017/05/curiousconvo_letterbox5b15d.jpg'
+  description: 'Rob is an artist based in South-east London and a Visitor Experience Assistant at Wellcome Collection. '+
+               'He also plays music in a ramshackle manner. He is a Gemini with a Libra ascendant.',
```

Comment:
in this case, both – either of them don't involve any computation, you just allocate a full String as a literal. But I'm being **hyper extra nitpicky** with this, it really has negligible impact unless you're running this on a microcontroller or something :)